src/terra-dev-site/contributing/GitWorkflow.b.contributing.md

```diff
@@ -0,0 +1,112 @@
+
+In order for your pull request to be accepted, the [Travis CI](https://travis-ci.org/) build needs to pass.
+
+### Small And Large Changes
+For small changes, at least one maintainer must approve the change. If a maintainer was the author of the change being reviewed, the approval must come from a different maintainer.
+
+Changes will be considered small unless otherwise noted by a label, “**Large**”, on the PR.
+
+For large, significant, or breaking changes, the participating maintainers must reach consensus on the change. Like *trivial*, the notion of *small* and *large* are subjective terms. However, a breaking change should be clear and unambiguous. Note that the size of the contribution does not determine whether the change requires just one approval or consensus. Rather, the impact of the change reflects the type of review required. For instance, the addition of a new component would be considered a significant change and would warrant consensus amongst the maintainers.
+
+Component API changes are scrutinized far more closely than that of documentation or related code projects (like sample projects, test harnesses, etc). As such, API changes are always considered significant and require consensus from the maintainers.
+
+Depending on the size of the changes, additional reviews via meetings/emails may be necessary.
+
+If decisions are made in meetings/emails, details/decisions should be added to the relevant GitHub issue as comments when possible to ensure that the decisions/changes/reasons are tracked and documented.
+
+### UX And Functional Reviews
+
+Along with reviews from maintainers, depending on the change, you may need a review from our **UX designer** and our **Solution designer**.
+
+This is denoted with the following labels added on the PR, “**Needs UX review**” and “**Needs functional review**”.
+
+Once our UX designer and Solution designer complete their reviews, they’ll leave a comment and remove the associated labels.
+
+### Tech Only Changes
+If a PR has a label of “**Tech Only**” on it, UX and functional review is not required, though this will still need reviews by maintainers based on the small and large change concepts.
+
+## PR Deployments
+When a PR is created in one of a the following repos, terra-core, terra-clinical, terra-framework, terra-ui, terra-dev-site, a Heroku PR deployment is triggered automatically that will generate a build of the code on the PR and deploy it to generated URL. This URL will be added as an link on the PR automatically based on the success of the Heroku build.
```

Comment:
want to link to those repos

Reply:
Updated: https://github.com/cerner/terra-ui/pull/108/commits/453b149ee491fa9a8d004c85dc550ca83dedf36d